pkg/naming/constants.go

```diff
@@ -19,6 +19,9 @@ const (
 	// ReplaceLabel express the intent to replace pod under the specific member.
 	ReplaceLabel = "scylla/replace"
 
+	// ReplacingNodeHostIDLabel contains the Host ID of node being replaced.
+	ReplacingNodeHostIDLabel = "internal.scylla-operator.scylladb.com/replacing-node-hostid"
```

Comment:
nit: inconsistent naming, ReplacingNode would mean a node which is replacing, not a node which is being replaced

Reply:
done

Reply:
How about `// ReplacingNodeHostIDLabel contains the Host ID of a node which the labelled node is replacing.` to improve readability?